### ISSUE TITLE
Problem about trainer_view.py

### ISSUE THREAD

**Issue by SpiritBear000** [pse-ecn/pose-sensitive-embedding#18]
hi,
I execute the following:
python3 trainer_views.py --output=everything_generate/output_dir --data=/media/lab210/Work/WH_HOME/SubjectDownload/Project/DATASETS/datasets/RAP/RAP_dataset --dataset-name=rap --batch-size=16 --num-epochs=100 --network-name=resnet_v1_50_views --initial-checkpoint=/media/lab210/Work/WH_HOME/SubjectDownload/Project/reid/pose-sensitive-embedding/everything_generate/checkpoint/resnet_v1_50.ckpt --checkpoint-exclude-scopes=everything_generate/resnet_v1_50/logits --trainable-scopes=resnet_v1_50/3Views --ignore-missing-variables

but get the error:
![2018-10-02 12-30-43](https://user-images.githubusercontent.com/32519071/46605051-c345da00-cb2a-11e8-9b48-6a1f120a2d23.png)

-----------------------------------------
And another question:

--checkpoint-exclude-scopes=everything_generate/resnet_v1_50/logits

where is the logits?
